saved-sites/saved-sites-impl/src/main/java/com/duckduckgo/savedsites/impl/service/SavedSitesParser.kt

```diff
@@ -34,7 +34,7 @@ interface SavedSitesParser {
     suspend fun parseHtml(
         document: Document,
         savedSitesRepository: SavedSitesRepository,
-    ): List<SavedSite>
+    ): List<Any>
```

Comment:
same here, find all common attributes used and create a data model for it. We shouldn’t return Any as part of public APIs

Reply:
Since it's not in the API module, I'm going to leave this one.
I will make a follow-up task to do a full refactor of `SavedSite` so that we can remove the use of `Any`.

Reply:
works for me!